### PR TITLE
raftstore/fsm/peer: use new log format

### PR DESCRIPTION
**Pull request by BusyJay**
## What have you changed? (mandatory)

Use the new log format for file fsm/peer.rs.

## What are the type of the changes? (mandatory)

- Improvement

## How has this PR been tested? (mandatory)

unit tests

## Does this PR affect documentation (docs) update? (mandatory)

No.

## Does this PR affect tidb-ansible update? (mandatory)

No.

### CODE REVIEW

File: src/raftstore/store/fsm/peer.rs [hunk 1]
```diff
@@ -155,9 +155,9 @@ impl PeerFsm {
     ) -> Result<(LooseBoundedSender<PeerMsg>, Box<PeerFsm>)> {
         // We will remove tombstone key when apply snapshot
         info!(
-            "[region {}] replicate peer with id {}",
-            region_id,
-            peer.get_id()
+            "replicate peer";
+            "region_id" => region_id,
+            "id" => peer.get_id(),
```
Comment (zhangjinpeng1987) [inline src/raftstore/store/fsm/peer.rs]: I prefer s/id/peer_id.

File: src/raftstore/store/fsm/peer.rs [hunk 2]
```diff
@@ -2372,9 +2529,19 @@ impl<'a, T: Transport, C: PdClient> PeerFsmDelegate<'a, T, C> {
     fn on_ready_compute_hash(&mut self, region: metapb::Region, index: u64, snap: EngineSnapshot) {
         self.fsm.peer.consistency_state.last_check_time = Instant::now();
         let task = ConsistencyCheckTask::compute_hash(region, index, snap);
-        info!("{} schedule {}", self.fsm.peer.tag, task);
+        info!(
+            "schedule task";
```
Comment (zhangjinpeng1987) [inline src/raftstore/store/fsm/peer.rs]: schedule compute hash task.

File: src/raftstore/store/fsm/peer.rs [hunk 3]
```diff
@@ -1713,9 +1815,10 @@ impl<'a, T: Transport, C: PdClient> PeerFsmDelegate<'a, T, C> {
         }
         if !stale {
             info!(
-                "{} merge to {:?} finish.",
-                self.fsm.peer.tag,
-                self.fsm.peer.pending_merge_state.as_ref().unwrap().target
+                "merge finish.";
```
Comment (zhangjinpeng1987) [inline src/raftstore/store/fsm/peer.rs]: s/finish/finished

File: src/raftstore/store/fsm/peer.rs (hunk 1 above)
Comment (zhangjinpeng1987) [reply src/raftstore/store/fsm/peer.rs]: Please ignore it since it in peer.rs.

Comment (breeswish) [reply src/raftstore/store/fsm/peer.rs]: Maybe still better to use peer_id to that it can be friendly to be searched in ElasticSearch later.

**Pull request merged**